File: exs/mundo_1/java/019.java
// Um professor quer sortear um dos seus quatro alunos para apagar o quadro.
// Faça um programa que ajude ele, lendo o nome deles e escrevendo o escolhido

import java.util.*;

class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        ArrayList<String> alunos = new ArrayList<>();

        System.out.print("Nome do aluno: ");
        alunos.add(scan.nextLine());

        System.out.print("Nome do aluno: ");
        alunos.add(scan.nextLine());

        System.out.print("Nome do aluno: ");
        alunos.add(scan.nextLine());

        System.out.print("Nome do aluno: ");
        alunos.add(scan.nextLine());

        int index = new Random().nextInt(alunos.size()); // select a index
        System.out.printf("\n%s vai apagar o quadro\n", alunos.get(index));
    }
}

File: exs/mundo_1/javascript/019.js
/*
Desafio 019

Problema: Um professor quer sortear um dos seus quatro alunos para 
          apagar o quadro. Faça um programa que ajude ele, lendo o 
          nome deles e escrevendo o do escolhido

Resolução do problema:
*/

const input = require('readline-sync').question

let nome1 = input('1º aluno(a): ')
let nome2 = input('2º aluno(a): ')
let nome3 = input('3º aluno(a): ')
let nome4 = input('4º aluno(a): ')

let indiceAleatorio = Math.floor(
    Math.random() * 4
)

let escolha = [nome1, nome2, nome3, nome4][indiceAleatorio]

console.log(`O aluno escolhido foi: ${escolha}`);